receiver/awscontainerinsightreceiver/README.md

```diff
@@ -443,6 +444,23 @@ kubectl apply -f config.yaml
 | Version            |
 | Sources            |
 
+
+### Cluster PriorityLevel
```

Comment:
Is Cluster PriorityLevel a new log event type? If its not, can we just keep this with the other apiserver metrics above? Or am i missing some reason this is separate?

Reply:
Spoke offline, moved all control-plane metrics to a single table